client/my-sites/purchases/navigation.tsx

```diff
@@ -21,9 +22,11 @@ export default function PurchasesNavigation( {
 			<NavTabs label="Section" selectedText={ sectionTitle }>
 				<NavItem
 					path={ `/purchases/subscriptions/${ siteSlug }` }
-					selected={ sectionTitle === 'Active Upgrades' }
+					selected={
+						isJetpackCloud() ? sectionTitle === 'Purchases' : sectionTitle === 'Active Upgrades'
```

Comment:
Is this comparison valid for translations?

Reply:
Thanks very much to both of you for reviewing. Yes good point here. I refactored the comparisons to use slug strings that map to the translations.